### PR TITLE
Update runtime.md

### PR DESCRIPTION
**Pull request by liangchenye**
Add an explicit explanation of the hooks order.
 
Signed-off-by: Liang Chenye <liangchenye@huawei.com>

### CODE REVIEW

**Comment by liangchenye** [comment]
Clean the original PR https://github.com/opencontainers/specs/pull/156..

**Comment by vbatts** [comment]
LGTM

File: runtime.md [hunk 1]
```diff
@@ -60,6 +61,7 @@ If a hook returns a non-zero exit code, then an error including the exit code an
 
 ### Post-stop
 The post-stop hooks are called after the container process is stopped. Cleanup or debugging could be performed in such a hook.
+
```
Comment (wking) [inline runtime.md]: Probably no need to add this empty line.

**Comment by philips** [comment]
lgtm

**Comment by vbatts** [comment]
@liangchenye rebase needed for merging

**Comment by liangchenye** [comment]
@vbatts done.

**Pull request merged**